### PR TITLE
Fixed Issue #372

### PR DESCRIPTION
**Pull request by Tu2-prog**
Informationen zu Transpersonen wurden hinzugefügt. Layoutfixes sind aber noch zu erledigen.

### CODE REVIEW

File: mathphys/arbeitsweise.tex [hunk 1]
```diff
@@ -23,7 +23,17 @@ \subsection{Arbeitskreise und Arbeitsgruppen}
...
 
 \vspace*{-2mm}
+\subsection{AK-Queer}
+In der Hochschulgruppe PR$\sqrt{-1}$DE treffen sich queere mathphysinfo Studis zum quatschen, vernetzen und gemeinsamen Feierabend feiern.
 
+Wir freuen uns über neue Gesichter im neuen Semester, komm doch einfach mal vorbei! Absprachen und Ankündigungen finden sich in der über den QR Code zugänglichen WhatsApp-Gruppe.
+
+Bei Fragen oder Ähnlichem kannst Du dich sonst auch an ak-queer@mathphys.info wenden.
+\begin{figure}[h]
+    \includegraphics*[width=\linewidth]{ak_queer_qr.png}
```
Comment (christian-heusel) [inline mathphys/arbeitsweise.tex:33]: The link in the qrcode is broken:
grafik

Why dont you just use QRCode Package:
```latex
....
\usepackage{qrcode}
...
\qrcode[height=8em, link]{https://mathphys.info/\~chris/studieninformationstag.pdf}
```

**Review by christian-heusel — CHANGES_REQUESTED** [review]

File: mathphys/arbeitsweise.tex (hunk 1 above)
Comment (Tu2-prog) [reply mathphys/arbeitsweise.tex:33]: QR - Code was given by the workshop. We do not know the concrete link in order to generate the QR-Code.

File: mathphys/arbeitsweise.tex [hunk 2]
```diff
@@ -23,7 +23,17 @@ \subsection{Arbeitskreise und Arbeitsgruppen}
...
 
 \vspace*{-2mm}
+\subsection{AK-Queer}
+In der Hochschulgruppe PR$\sqrt{-1}$DE treffen sich queere mathphysinfo Studis zum quatschen, vernetzen und gemeinsamen Feierabend feiern.
 
+Wir freuen uns über neue Gesichter im neuen Semester, komm doch einfach mal vorbei! Absprachen und Ankündigungen finden sich in der über den QR Code zugänglichen WhatsApp-Gruppe.
+
+Bei Fragen oder Ähnlichem kannst Du dich sonst auch an ak-queer@mathphys.info wenden.
+\begin{figure}[h]
+    \includegraphics*[width=\linewidth]{ak_queer_qr_fixed.png}
```
Comment (christian-heusel) [inline mathphys/arbeitsweise.tex:33]: I would remove the broken one and rename this to just `ak_queer.png`

**Review by christian-heusel — APPROVED** [review]
Now the link works 🥳

**Pull request merged**